codegen/smithy-dafny-codegen/src/main/java/software/amazon/polymorph/smithygo/localservice/shapevisitor/DafnyToSmithyShapeVisitor.java

```diff
@@ -178,9 +178,8 @@ public String listShape(ListShape shape) {
 			if !ok {
 				break
 			}
-			fieldValue = append(fieldValue, %s%s)}
+			fieldValue = append(fieldValue, %s)}
```

Comment:
What's the reasoning behind structure not being a pointer??

Reply:
This is inside listshape. Anything inside listshape is a non pointer similar to map

Reply:
The question is more like can list have pointer. I have reason for both yes and no. But move forward with current code and revisit this if needed.